<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_nil?' type='fail'>
    <section_ name='Args=[nil, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_nil?' outcome='failed' id='0' message='Message 0'>
        <actual_ class='NilClass' value='nil'/>
        <exception_ class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[nil, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_nil?' outcome='failed' id='1' message='Message 1'>
        <actual_ class='NilClass' value='nil'/>
        <exception_ class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
